publishing-bot/manifests/secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: github-token
+  namespace: publishing-bot
+data:
+  token: "TOKEN"
```

Comment:
@dims @nikhita I'm not familiar with git-crypt. How do I can recreate the token in this repo ?

Reply:
@ameukam I don't think we prefer storing the token here. it should be where it is now in publishing-bot repo. @nikhita @sttts or i would then upload the token to the cluster would be better i think. will let them chime in. we just have a script or something that will help the 3 of us upload the token (sorry i haven't looked at the rest of the PR and just responding to this comment)

Reply:
Agree with @dims' comment. The token has admin access over multiple repos, so we'd want to restrict access to it by keeping it in the publishing-bot repo only.